data/ecosystems/s/solana.toml

```diff
@@ -48254,6 +48256,9 @@ url = "https://github.com/Grim-R3ap3r/Solana-Project"
 [[repo]]
 url = "https://github.com/Grim-Syndicate/eta-backend"
 
+[[repo]]
+url = "https://github.com/Grim-Syndicate/eta-frontend"
```

Comment:
this line can be removed, out ETL takes care of rolling all up

Reply:
and done